src/main/java/Whisper.java

```diff
@@ -0,0 +1,238 @@
+import java.util.*;
```

Comment:
might be good to use specific import packages to fulfil the coding style rule :)

Reply:
noted! thanks